backend/src/graphql/resolvers/RoomResolver.ts

```diff
@@ -95,6 +95,23 @@ export class RoomResolver {
     })
   }
 
+  @Query(() => String, { nullable: true })
+  async joinRoom(
+    @Arg('userName') userName: string,
+    @Arg('roomId', () => Int) roomId: number,
+  ): Promise<string | null> {
+    const meeting = await prisma.meeting.findUnique({
+      where: {
+        id: roomId,
+      },
+    })
+    if (!meeting) return null
```

Comment:
Like comment before I would prefer to get an error instead of null for this senario.
```suggestion
    if (!meeting) throw new Error('Room not found.')
```